packages/connectivity_plus/connectivity_plus/darwin/connectivity_plus.podspec

```diff
@@ -14,12 +14,22 @@ Downloaded by pub (not CocoaPods).
   s.author           = { 'Flutter Community Team' => 'authors@fluttercommunity.dev' }
   s.source           = { :http => 'https://github.com/fluttercommunity/plus_plugins/tree/main/packages/connectivity_plus' }
   s.documentation_url = 'https://pub.dev/packages/connectivity_plus'
-  s.source_files = 'Classes/**/*'
-  s.public_header_files = 'Classes/**/*.h'
-  s.dependency 'Flutter'
+  s.source_files = [
+    'Classes/**/*'
+  ]
+  s.public_header_files = [
+    'Classes/**/*.h'
+  ]
+  s.ios.dependency 'Flutter'
+  s.osx.dependency 'FlutterMacOS'
   s.dependency 'ReachabilitySwift'
-  s.platform = :ios, '12.0'
+  s.platform = {
+    :ios => '12.0',
+    :osx => '10.14'
+  }
+  s.ios.deployment_target = '12.0'
+  s.osx.deployment_target = '10.14'
```

Comment:
Do you know if it is necessary to define both `platform` and `deployment_target`? The docs only mention `deployment_target`: https://docs.flutter.dev/packages-and-plugins/developing-packages#shared-ios-and-macos-implementations